features/step_definitions/metadata.js

```diff
@@ -81,9 +74,9 @@ module.exports = function server() {
     );
 
     this.Then(/^a record of the metadata is stored$/, { timeout: TIMEOUT }, () => {
-        Assert.ok('foo', this.meta);
-        Assert.ok('bar', this.meta);
-        Assert.equal('foobar', this.meta.foo);
-        Assert.equal('barbaz', this.meta.bar);
+        Object.keys(this.expectedMeta).forEach((key) => {
+            Assert.ok(key, this.meta);
```

Comment:
I do not think this is doing what you think it is doing. http://chaijs.com/api/assert/#method_isok

For one, chai.assert does not have a documented `ok` method. This is borrowed from `expect` so it still works (for now). The documented api for chai.assert is `Assert.isOk(value, message)` where it determines if `value` is truthy. In this loop, `key` is always truthy. All you are doing here is verifying that the keys of expectedMeta are defined.

What I suspect you are actually trying to to is `Assert.property(this.meta, key)` to determine if the actual meta object has a property with the key from the value of `key`. 

This really isn't necessary either, since the next line asserts that there is a value at that key in actual meta that matches an expected value with the same key. If the key didn't exist, the value would be `undefined`, and that will not match the expected value.